2025-02-19 19:22:36,443 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:23:36,446 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:24:36,448 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:25:36,451 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:25:59,805 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:26:59,830 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:27:53,485 - ERROR - Žádná validní data nebyla nalezena: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:39:32,812 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:40:18,909 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:41:48,107 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:43:50,574 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:50:08,230 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:52:55,988 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:53:56,197 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:54:56,401 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:55:56,621 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:56:56,834 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_PAVEL.tch
2025-02-19 19:56:56,835 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:57:57,061 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_PAVEL.tch
2025-02-19 19:57:57,062 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:58:57,288 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_PAVEL.tch
2025-02-19 19:58:57,290 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 19:59:57,527 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_PAVEL.tch
2025-02-19 19:59:57,528 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 20:00:57,766 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_PAVEL.tch
2025-02-19 20:00:57,767 - ERROR - Prázdné nebo chybějící user_name: /Users/mpmp/Downloads/10mV57a/Match_.tch
2025-02-19 20:06:29,688 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_ondrejzaplatilek.tch
2025-02-19 20:15:53,722 - ERROR - Prázdný nebo chybějící GameData element: /Users/mpmp/Downloads/10mV57a/Match_ondrejzaplatilek.tch
2025-02-20 16:40:57,226 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:29,629 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:30,878 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:32,159 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:33,380 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:34,577 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:35,839 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:37,114 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:38,367 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:39,612 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:40,819 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:42,057 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:43,259 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:44,538 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:45,779 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:47,028 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 16:41:48,233 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 17:14:40,778 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
2025-02-20 17:14:42,910 - ERROR - API Communication Error: HTTPSConnectionPool(host='shooting.bytedev.cz', port=443): Max retries exceeded with url: /api/shots (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1000)')))
